`npm run docs` failing locally for build process

`npm run docs` does not work locally on Windows operating systems. When `npm install` is run on a fresh project, the binary file for JSDocs is not initialized under `node_modules/.bin/`. 

`'jsdoc' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cse110-w21-group7@1.0.0 docs: ``jsdoc source/javascript/src/* --destination specs/ci-cd/docs/``
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cse110-w21-group7@1.0.0 docs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.`

--- a/package.json
+++ b/package.json
@@ -4,13 +4,13 @@
   "description": "RIVER Bullet Journal; CSE110 Spring 2021",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --recursive ./source/javascript/tests/*",
+    "test": "mocha --recursive ./source/javascript/tests/.",
     "coverage": "nyc npm run test",
-    "lint_tests": "./node_modules/.bin/eslint source/javascript/tests/*",
-    "fix_tests": "./node_modules/.bin/eslint source/javascript/tests/* --fix",
-    "lint_src": "./node_modules/.bin/eslint source/javascript/src/*",
-    "fix_src": "./node_modules/.bin/eslint source/javascript/src/* --fix",
-    "docs": "./node_modules/.bin/jsdoc source/javascript/src/* --destination specs/ci-cd/docs/"
+    "lint_tests": "./node_modules/.bin/eslint source/javascript/tests/.",
+    "fix_tests": "./node_modules/.bin/eslint source/javascript/tests/. --fix",
+    "lint_src": "./node_modules/.bin/eslint source/javascript/src/.",
+    "fix_src": "./node_modules/.bin/eslint source/javascript/src/. --fix",
+    "docs": "npx jsdoc source/javascript/src/. --destination specs/ci-cd/docs/"
   },
   "repository": {
     "type": "git",

